src/libs/actions/Session/index.js

```diff
@@ -392,7 +392,7 @@ function clearAccountMessages() {
     Onyx.merge(ONYXKEYS.ACCOUNT, {
         error: '',
         success: '',
-        errors: [],
+        errors: null,
```

Comment:
oh maybe it doesn't matter all that much, I see elsewhere in the file we set `errors` to `null`.

Reply:
It's set to null here(https://github.com/Expensify/App/pull/10556/files#diff-571033c4e40d8aa5e2b60377b898d82d804e539b987033e6979e89352a567b8aR98), I realized some console errors disappeared when it was null instead since the [DotIndicatorMessage](https://github.com/Expensify/App/pull/10556/files#diff-ed15e2a2b3ba051db0688c6e465dae1d08147a3dabaa4bead5cb6b7f9c361f76R20) expect an object and not an array.